src/components/avatar/index.tsx

```diff
@@ -44,6 +46,35 @@ export enum BadgePosition {
 
 const DEFAULT_BADGE_SIZE = 'pimpleBig';
 
+export type NameToColorProps = {
+  /**
+   * Hash the name (or label) to get a color, so each name will have a specific color.
+   * Default is false.
+   */
+  useHashBackgroundColor: boolean;
+  /**
+   * Avatar colors to be used when useHashBackgroundColor is true
+   */
+  avatarColors?: string[];
+  /**
+   * Replace the default hashing function (name -> number)
+   */
+  hashFunction?: (name?: string) => number;
+  /**
+   * Background color in cases where the getBackgroundColor returns undefined.
+   */
+  defaultColor?: string;
+  /**
+   * Replace the default logic.
+   */
+  getBackgroundColor?: (
```

Comment:
same as `getInitials`, if the user wants to have a custom logic for the background color they can implement on their end and pass thw `backgroundColor` prop

Reply:
Removed